src/Symfony/Component/Serializer/Normalizer/PropertyNormalizer.php

```diff
@@ -96,9 +93,7 @@ public function denormalize($data, $class, $format = null, array $context = arra
         $object = $this->instantiateObject($data, $class, $context, $reflectionClass, $allowedAttributes);
 
         foreach ($data as $propertyName => $value) {
-            if ($this->nameConverter) {
-                $propertyName = $this->nameConverter->denormalize($propertyName);
-            }
+            $propertyName = $this->normalizeName($propertyName);
```

Comment:
Can you verify in the tests if this is still working? You're now calling normalize instead of denormalize.